jetty-util/src/main/java/org/eclipse/jetty/util/resource/PathResource.java

```diff
@@ -93,9 +92,11 @@ private final Path checkAliasPath()
         }
 
         if (!abs.isAbsolute())
-        {
             abs = path.toAbsolutePath();
-        }
+
+        Path normal = path.normalize();
+        if (!abs.equals(normal))
+            return normal;
```

Comment:
What happens if the normalized path is to a symlink? Do we want to do the code below to resolve the target of the symlink?